### PR TITLE
[T3A4][T10-A2] Low Yong Cheng

### PR DESCRIPTION
**Pull request by yongchenglow**
Ignore the other 2 files, I updated the master yesterday so the Address book is manipulated already. The Storage File is where the Excepted is added

### CODE REVIEW

**Comment by yongchenglow** [comment]
Ready for review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,18 +11,18 @@
 public class CommandResult {
```
Comment (nirandiw) [inline src/seedu/addressbook/commands/CommandResult.java]: Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -8,12 +8,17 @@
  */
 public class Address {
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Address.java]: Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/storage/StorageFile.java [hunk 3]
```diff
@@ -101,6 +105,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         } catch (JAXBException jaxbe) {
             throw new StorageOperationException("Error converting address book into storage format");
         }
+        
+        if(!fileExist){
+        	throw new StorageOperationException("Warning, your storage file may have been deleted, the current storage file has recovered what we could");
+        }
```
Comment (nirandiw) [inline src/seedu/addressbook/storage/StorageFile.java]: - Good job in throwing the exception. But you have not handled it. You can not just throw exceptions and let someone else deal with it.

- Implement how to catch and handle this exception. Hint: Main.java executeCommand

**Comment by nirandiw** [comment]
@yongchenglow Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**